install/nightly/InstallHalyard.sh

```diff
@@ -1,427 +1,23 @@
 #!/usr/bin/env bash
 
-# This script installs Halyard.
+TEMPDIR=$(mktemp -d installhalyard.XXXX)
 
-# _WARNING_ This file was auto generated, do not edit directly
+pushd $TEMPDIR
 
-set -e
-set -o pipefail
+echo "Downloading nightly halyard installer..."
 
-REPOSITORY_URL="https://dl.bintray.com/spinnaker-releases/debians"
-SPINNAKER_REPOSITORY_URL="https://dl.bintray.com/spinnaker-releases/debians"
-SPINNAKER_DOCKER_REGISTRY="gcr.io/spinnaker-marketplace"
-SPINNAKER_GCE_PROJECT="marketplace-spinnaker-release"
+curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/Installer.sh
```

Comment:
Seems like we would need to produce the Halyard JARs and publish them to GCS before we flip this switch, correct? Moreover, are we preserving the old method of installing Halyard for folks who aren't ready to migrate yet?